### ISSUE TITLE
Consider using venv instead of virtualenv

### ISSUE THREAD

**Issue by RaddishIoW** [aws/aws-elastic-beanstalk-cli-setup#126]
Python includes a virtual environment package as part of the base installation - "venv".

Ideally, I do not want to install virtualenv as I do not use it for any other purpose, preferring to use the built-in venv package.

Would it be a good idea to allow the use of venv to install and use EBCLI instead of requiring a 3rd party package?

**Comment by Suncatcher** [comment]
Agree. It makes no sense to use deprecated tool like `virtualenv`. Venv is a stanard and has the same capabilities, the only reason people use `virtualenv` is backward compatibility with Python2.

**Comment by fire-space** [comment]
I second this. I came here hoping this was a planned update
